_drafts/early-stage/SC4-1-2-programmatic-name.md

```diff
@@ -0,0 +1,82 @@
+---
+rule_id: SC4.1.2-Name-Role-Value-id
```

Comment:
I think the identifier here is intended to match the file name and has a specific syntax. The WCAG ref SC4-1-2 is sufficient for association with a success criteria, the rest should identify the test rule itself.  SC4-1-2-programmatic-name seems fine.